config/tags.php

```diff
@@ -197,7 +198,20 @@
 				Uuid::is($tag->value, 'page') === true ||
 				Uuid::is($tag->value, 'file') === true
 			) {
-				$tag->value = Uuid::for($tag->value)->model()->url();
+				$tag->value = Uuid::for($tag->value)->model()?->url();
+			}
+
+			// if url is empty, throw exception or link to the error page
+			if ($tag->value === null) {
+				if ($tag->kirby()->option('debug', false) === true) {
+					if (empty($tag->text) === false) {
+						throw new NotFoundException('The linked page cannot be found for the link text "' . $tag->text . '"');
+					} else {
+						throw new NotFoundException('The linked page cannot be found');
+					}
+				} else {
+					$tag->value = $tag->kirby()->site()->errorPage()->url();
```

Comment:
It only happens if we don't have an error page, but I think this is why the tests fail. 

```suggestion
					$tag->value = $tag->kirby()->site()->errorPage()?->url();
```

Reply:
What about `Url::to($tag->kirby()->site()->errorPageId())`?

Reply:
Good idea.